common/printfile.cpp

```diff
@@ -256,7 +256,8 @@ void Output::print_local_file(const std::string& data) {
 
 bool Output::printfile_random(const std::string& data) {
   const printfile_opts opts(sess(), *this, data, true, true);
-  const auto& dir = sess().dirs().current_menu_gfiles_directory();
+//  const auto& dir = sess().dirs().current_menu_gfiles_directory();
```

Comment:
what will this break?

Reply:
you can nix this. currently printfile_random and printfile default to using gfiles/menus/wwiv/gfiles/, which permits multiple menu configs. i was simplifying to just use gfiles/ directly.

Reply:
what do you think the right behavior here is?

Reply:
check the current_menu_gfiles_directory() first if the file exists there, then check gfiles directory... i'll rework this.